docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -16,21 +16,24 @@ The User Feedback Widget offers many customization options, and if the available
+| `id`           | `string`                                   | `sentry-feedback` | The `id` attribute of the `<div>` that contains the feedback widget. See [Theme Customization](#theme-customization) for more.                                                                                                           |
 
 ### User and Form
 
-| key               | type                     | default                               | description                                                                                                                 |
-| ----------------- | ------------------------ | ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| `showName`        | `boolean`                | `true`                                | Displays the name field on the Feedback form, however will still capture the name (if available) from Sentry SDK context.   |
-| `showEmail`       | `boolean`                | `true`                                | Displays the email field on the Feedback form, however will still capture the email (if available) from Sentry SDK context. |
-| `isNameRequired`  | `boolean`                | `false`                               | Requires the name field on the Feedback form to be filled in.                                                               |
-| `isEmailRequired` | `boolean`                | `false`                               | Requires the email field on the Feedback form to be filled in.                                                              |
-| `useSentryUser`   | `Record<string, string>` | `{ email: 'email', name: 'username'}` | Map of the `email` and `name` fields to the corresponding Sentry SDK user fields that were called with `Sentry.setUser`.    |
+The following options can be configured for the integration in `feedbackIntegration({})`:
+
+| key               | type                     | default                               | description                                                                                                                    |
+| ----------------- | ------------------------ | ------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `showName`        | `boolean`                | `true`                                | Displays the name field on the Feedback form, however will still capture the name (if available) from Sentry SDK context.      |
+| `showEmail`       | `boolean`                | `true`                                | Displays the email field on the Feedback form, however will still capture the email (if available) from Sentry SDK context.    |
```

Comment:
"However will still capture the name/email" is confusing to me. Does this mean the name/email is captured regardless of whether this option is true or false? Or does it mean that even if the name field is shown, the name is still retrieved from SDK context (in this case, is it stored separately?)? Can you clarify what the behavior is?

Reply:
I'm making the change where the table will only say: "Displays the name field on the Feedback form."

and below there'll be two paragraphs, with the `setUser()` example below:
>#### User Context
>
> If you have set a user context by calling [Sentry.setUser](https://docs.sentry.io/platforms/javascript/enriching-events/identify-user/) then those values will be used as defaults for the name and email fields. If those fields are hidden from the user, then the default values will still be sent along with the feedback message.
>
> Below is an example configuration with non-default user fields.

It'll be up here in a sec, but also lmk if that's working better!